background.js

```diff
@@ -202,3 +202,25 @@ chrome.storage.local.get('version', prefs => {
 		}
 	}
 });
+
+chrome.runtime.onInstalled.addListener(({reason, previousVersion, id} = {}) => {
```

Comment:
If we execute it as is it'll inject a second content script in normal cases. I'll add the code that checks if the content script is alive first.